hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/connector/file/FileTableFunction.java

```diff
@@ -83,6 +94,43 @@ private static String randomName() {
         return SCHEMA_NAME_FILES + "_" + UuidUtil.newUnsecureUuidString().replace('-', '_');
     }
 
+    @Override
+    public List<Permission> permissions(SqlCall call) {
+        SqlNode astPath = findOperandByName(OPTION_PATH, call);
```

Comment:
arguments can be passed also by position and by options ( ? not sure about this one), shouldn't we check those as well?

Reply:
By options - no.
By position is checked (1st condition).

Reply:
https://github.com/hazelcast/hazelcast/pull/25784/commits/23a9065f45c9400a86631c4e99214e12ff96ea11

Reply:
https://github.com/hazelcast/hazelcast/commit/23a9065f45c9400a86631c4e99214e12ff96ea11